app.py

```diff
@@ -4,44 +4,78 @@
 import json
 from flask import Flask, abort, jsonify, render_template, request, send_file
 from flask_cors import CORS, cross_origin
-from generate_polygon import generate_polygon
 from generate_set import generate_sets
-
+from generate_points import generate_points
+from voronoi_gen import generate_voronoi
+from convex_hull_gen import convex_hull_gen
 # Get the parent directory of this script. (Global)
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
 app = Flask(__name__)
 CORS(app)
 
-@app.route('/' , methods=["GET"])
+
+@app.route('/', methods=["GET"])
 def index():
     return "Hello World"
 
-@app.route('/',methods=["POST"])
+
+@app.route("/generate_points_voronoi/", methods=["POST"])
+@cross_origin()
+def get_data_voronoi():
+    card = int(request.get_json()['card'])

```

Comment:
Try to keep a limit on the card input as 10..
Ask the user to use MBR based stuff